scripts/geofence_email_notifications.py

```diff
@@ -0,0 +1,128 @@
+"""
+   Copyright 2020 Esri
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.​
+
+"""
+import argparse
+import logging
+import logging.handlers
+import traceback
+import pendulum
+import sys
+import yagmail
```

Comment:
I would prefer we not use a third party library and just use the standard library, it seems simple enough.